src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -242,11 +247,23 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
      */
     public raySelectionPredicate: (mesh: AbstractMesh) => boolean;
 
-    /** filter used for near interaction pick and hover */
-    public nearPickPredicate(mesh: AbstractMesh): boolean { return mesh.isEnabled() && mesh.isVisible && mesh.isPickable && mesh.isNearPickable};
-    
-    /** filter used for near interaction grab */
-    public nearGrabPredicate(mesh: AbstractMesh): boolean {return mesh.isEnabled() && mesh.isVisible && mesh.isPickable && mesh.isNearGrabbable};
+    /**

```

Comment:
Hm. I don't think the predicate functions here can be locally overwritten in this setup. If we want to allow developers to use their own predicates for near interactions, I think it needs to be set up like so:
public nearGrabPredicate = (...)=>{...};

Otherwise if this stays as a function, there's probably no need to publicly expose it.

Reply:
That is actually true. I  will make this private